content/en/guide/v10/typescript.md

```diff
@@ -56,13 +57,13 @@ If you use TypeScript within a Babel toolchain, set `jsx` to `preserve` and let
 
 Rename your `.jsx` files to `.tsx` for TypeScript to correctly parse your JSX.
 
-## Typing components
+## Typing Components
 
-There are different ways to type components in Preact. Class components have generic type variables to ensure type safety. TypeScript sees a function as functional component as long as it returns JSX. There are multiple solutions to define props for functional components.
```

Comment:
> TypeScript sees a function as functional component as long as it returns JSX.

Unless a function is matched `FunctionComponent` or `FunctionalComponent`, it is not considered a functional components.
This is incorrect.

Reply:
I think this line is unnecessary.

Reply:
Since there is the same content at below, I think this line is unnecessary.